src/pytest_plugins/forks/forks.py

```diff
@@ -345,6 +355,12 @@ def get_validity_marker_args(
             f"'{validity_marker_name}'. "
             f"List of valid forks: {', '.join(metafunc.config.fork_names)}"  # type: ignore
         )
+    fork_folder = metafunc.module.__name__.split(".")[1].capitalize()
+    if globals()[fork_name] < globals()[fork_folder]:
+        pytest.fail(
+            f"Incorrect usage of '{validity_marker_name}'. The validity marker fork "
+            f"'{fork_name}' must be greater than or equal to the test folder fork '{fork_folder}'."
+        )
```

Comment:
Validity marker check. Note this required us to move some existing tests or change the fork name.